Makefile

```diff
@@ -23,7 +23,7 @@ crio:
 	bash .ci/install_bats.sh
 	cp $(PWD)/integration/cri-o/crio.bats ${CRIO_REPO_PATH}/test/
 	cd ${CRIO_REPO_PATH} && \
-	RUNTIME=${CC_RUNTIME} ./test/test_runner.sh crio.bats
+	RUNTIME=${CC_RUNTIME} STORAGE_OPTS="--storage-driver=devicemapper" ./test/test_runner.sh crio.bats
```

Comment:
A couple of questions:

- Should the storage driver be logged with an `echo` here for clarity?
- Should the tests in fact be run twice, once for each storage driver?

Reply:
@jodh-intel yes we can echo in order to log which driver is in use.
About running twice, we had this conversation with @chavafg and @sameo and we agreed that it would take lot of time to test both overlay and devicemapper for every distro. Instead, we thought that having one distro testing devicemapper and another one testing overlay could be a good tradeoff

Reply:
@sboeuf - great. Sorry, I'm confused - spreading the drivers across the O/S's sounds fine to me too, but this file is `Makefile` is generic isn't it?

Reply:
Yes yes you're right, this PR was only to make a test. If this now works properly, we're gonna need a bit more logic to achieve this split testing between distros.